pod-operation/src/components/brakes.rs

```diff
@@ -0,0 +1,27 @@
+use tracing::debug;
+
+use rppal::gpio::{Gpio, OutputPin};
+
+pub struct Brakes {
+	pin: OutputPin,
+}
+
+const PIN_BRAKES: u8 = 5; //Replace with real value
```

Comment:
Chore: the actual relay pin number we're using should be 26 based on last year

Reply:
perf